Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -2496,8 +2496,6 @@ class AlignFunctionDefinitions(CythonTransform):
 
     def visit_ModuleNode(self, node):
         self.scope = node.scope
-        self.directives = node.directives
-        self.imported_names = set()  # hack, see visit_FromImportStatNode()
```

Comment:
?

Reply:
I think this is ok - these two attributes are just moved to `AutoCpdefFunctionDefinitions` (because they're only used for that).

The rest of the "?"s I agree with